conformance/utils/suite/suite.go

```diff
@@ -412,6 +412,10 @@ func (suite *ConformanceTestSuite) Run(t *testing.T, tests []ConformanceTest) er
 			test:   test,
 			result: res,
 		}
+		if res == testSucceeded || res == testFailed {
```

Comment:
Nit: Would be good to skip this step when you're finishing the last step - presumably it's only useful in between tests and not after the last one.

Reply:
Done.

Reply:
+1. The sleep is performed for the last test as well. We should perform some checks on the test index.

Reply:
Thanks for the comments. I've updated the code here to sleep before running each test except the first test case. Please review again